lib/msal-angular/docs/multi-tenant.md

```diff
@@ -45,3 +45,46 @@ export class AppComponent implements OnInit {
     });
   }
 ```
+
+## Dynamic auth request
+
+By default the MsalGuard and the MsalInterceptor use the static properties you set in the config. Both can be configured with a method for the `authRequest`. This allows to dynamically change the parameters used for authentication.
+
+### MsalInterceptor - dynamic auth request (multi tenant tokens)
+
+If you're using `organizations` or `common` all the tokens will be requested for the users' home tenant, this might not result in the desired effect. If you have this application where you're invited as a guest the tokens will be from the wrong authority.
```

Comment:
```suggestion
If `organizations` or `common` is used as the tenant, all tokens will be requested for the users' home tenant. However, this may not be the desired outcome. If a user is invited as a guest, the tokens may be from the wrong authority.
```